src/main.rs

```diff
@@ -7,6 +7,7 @@
 
 #![warn(missing_docs)]
 #![warn(missing_debug_implementations)]
+#![warn(clippy::rc_buffer)]
```

Comment:
why do you need this? since its included in `all`

Reply:
maybe we don't.  i'll remove it again